docs/shared/helm-show-router-output.mdx

```diff
@@ -0,0 +1,238 @@
+```yaml
+Pulled: ghcr.io/apollographql/helm-charts/router:1.31.0
+Digest: sha256:26fbe98268456935cac5b51d44257bf96c02ee919fde8d47a06602ce2cda66a3
+# Default values for router.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+# -- See https://www.apollographql.com/docs/router/configuration/overview/#yaml-config-file for yaml structure
+router:
+  configuration:
+    supergraph:
+      listen: 0.0.0.0:80
+    health_check:
+      listen: 0.0.0.0:8088
+    telemetry:
+      metrics:
+        prometheus:
+          enabled: false
+          listen: 0.0.0.0:9090
+          path: "/metrics"
+
+  args:
+  - --hot-reload
+
+managedFederation:
+  # -- If using managed federation, the graph API key to identify router to Studio
+  apiKey:
+  # -- If using managed federation, use existing Secret which stores the graph API key instead of creating a new one.
```

Comment:
```suggestion
  # -- If using managed federation, use an existing secret which stores the graph API key instead of creating a new one.
```

Reply:
@garypen - a note to incorporate the copyedits from this PR into both the release and in-house router charts